docs/spelling_wordlist.txt

```diff
@@ -157,3 +157,4 @@ PathLike
 ElementType
 customizable
 tokenizes
+param
```

Comment:
I don't see `param` being used anywhere in the PR. Is there any reason to add it to the wordlist?